src/main/java/Main.java

```diff
@@ -1,8 +1,32 @@
-public class Main {
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
 
+public class Main {
 
     public static void main(String[] args) {
 
-    }
+        try {
+            Scanner scanner = new Scanner(new File("liczby.txt"));
```

Comment:
Pamiętaj o zamykaniu pliku, albo najlepiej wprowadź try-with-resources https://youtu.be/LRwSHA9YDgY